tox.ini

```diff
@@ -1,7 +1,14 @@
 [tox]
-envlist = py27
+envlist = py27,flake8
 
 [testenv]
 deps = -r{toxinidir}/test-requirements.txt
 commands =
     nosetests --tests tests
+
+[testenv:flake8]
+commands = flake8
+
+[flake8]
+show-source = True
+max-line-length = 160
```

Comment:
to what value? 120? I tough 160 was reasonable but I'm flexible enough to have a shorter line if you have a specific number in you're head